src/index.js

```diff
@@ -1,7 +1,55 @@
 // Importing CSS styles
 import './style.css';
-import render from './modules/displaylist.js';
+import {
+  render,
+  addTask,
+  editTask,
+  removeTask,
+} from './modules/handletodo.js';
 
-// Adding main component to the HTML file
-const mainContainer = document.querySelector('.main');
-mainContainer.appendChild(render());
+render();
+// add task when text is submitted
+const taskinput = document.querySelector('.task-desc');
```

Comment:
- [OPTIONAL] As suggested by the previous reviewer, kindly ensure that your page does not refresh at every operation you perform as to give the best user experience.